opencti-platform/opencti-graphql/src/modules/case/case.ts

```diff
@@ -20,7 +20,7 @@ const CASE_DEFINITION: ModuleDefinition<StoreEntityCase, StixCase> = {
   },
   identifier: {
     definition: {
-      [ENTITY_TYPE_CONTAINER_CASE]: [{ src: NAME_FIELD }, { src: 'case_type' }, { src: 'created' }]
+      [ENTITY_TYPE_CONTAINER_CASE]: [{ src: NAME_FIELD }, { src: 'created' }]
```

Comment:
I am not sure thant's a good idea to register this entity like the other, because it's like an interface.